service/lib/dinstaller/dbus/service.rb

```diff
@@ -32,7 +32,7 @@ module DInstaller
   module DBus
     # D-Bus service (org.opensuse.DInstaller)
     #
-    # It connects to the system D-Bus and answers requests on objets below
+    # It connects to the system D-Bus and answers requests on objects below
     # `/org/opensuse/DInstaller`.
     class Service
```

Comment:
From now on, due to we are going to provide different services, I would use more specific service names, for example: `DInstaller::DBus::Services::Manager`.

Reply:
Well, it will make sense when we provide only manager there like probe, install and status. But for now it contains much more methods that is not just manager one.